sections/collage.liquid

```diff
@@ -161,6 +161,26 @@
       "default": "Multimedia collage",
       "label": "t:sections.collage.settings.heading.label"
     },
+    {
+      "type": "select",
+      "id": "heading_size",
+      "options": [
+        {
+          "value": "h2",
+          "label": "t:sections.all.heading_size.options__1.label"
+        },
+        {
+          "value": "h1",
+          "label": "t:sections.all.heading_size.options__2.label"
+        },
+        {
+          "value": "h0",
+          "label": "t:sections.all.heading_size.options__3.label"
+        }
+      ],
+      "default": "h1",
```

Comment:
Is the default in general supposed to be `h1` ? We used to have `h2` as a default before which is why I ask if the change is intentional or not.